travvy/src/components/OnboardingPage.vue

```diff
@@ -0,0 +1,154 @@
+<template>  
+  <div class="Onboarding">
+     <div class="container">
+    <!-- Onboarding fields that the user must fill in in order to get personalized recommendations --> 
+    <h2>Welcome to Travvy, we'd love to learn more about you!</h2>
+    <p>Please fill in the following information in order to get a custom travel experience.</p>
+    <label>Please select the sex you identify with: </label>
+    <br>
+    <br>
+    <div class="custom-select">
+    <select>
+        <option></option>
+        <option>Male</option>
+        <option>Female</option>
+        <option>Prefer not to say</option>
+    </select>
+    <br>
+    <br>
+    <label>Please input your age: </label>
+    <input type="text" > 
+    <br>
+    </div>
+    <br>
+    <label>Please select the type of activity you enjoy most: </label>
+    <br>
+    <br>
+    <button v-on:click="reverseMessage" class="nextActivity">Family</button>   <button v-on:click="reverseMessage" class="nextActivity">Historical</button>   <button v-on:click="reverseMessage" class="nextActivity">Active</button>  <button v-on:click="reverseMessage" class="nextActivity">Nature</button>  <button v-on:click="reverseMessage" class="nextActivity"> Entertainment</button> <button v-on:click="reverseMessage" class="nextActivity">Food</button> 
```

Comment:
This 'reverseMessage' on click is not necessary, we can remove this in future sprint

Reply:
Great catch, I'll keep that in mind, I was playing around with the buttons and forgot to delete the unnecessary code. Thanks, Rushan!!